components/brave_wallet/browser/nft_metadata_fetcher.cc

```diff
@@ -290,6 +290,7 @@ void NftMetadataFetcher::GetSolTokenMetadata(
       base::BindOnce(&NftMetadataFetcher::OnGetSolanaAccountInfoTokenMetadata,
                      weak_ptr_factory_.GetWeakPtr(), std::move(callback));
   json_rpc_service_->GetSolanaAccountInfo(*associated_metadata_account,
+                                          mojom::kSolanaMainnet,
```

Comment:
This drops support of getting metadata for devnet and testnet if I understand correctly.
I think we should instead add chain_id param to `GetSolTokenMetadata` API and pass it along to make the request to the right chain, we do that for eth chains too.